3scaleAdapter/pkg/threescale/proxy_conf.go

```diff
@@ -0,0 +1,108 @@
+package threescale
+
+import (
+	"fmt"
+	"sync"
+	"time"
+
+	sysC "github.com/3scale/3scale-porta-go-client/client"
+	"github.com/3scale/istio-integration/3scaleAdapter/config"
+
+	"istio.io/istio/pkg/log"
+)
+
+// DefaultCacheTTL - Default time to wait before purging expired items from the cache
+const DefaultCacheTTL = time.Duration(time.Minute * 1)
+const cacheKey = "%s-%s"
+
+type proxyStore struct {
+	element sysC.ProxyConfigElement
+	setAt   time.Time
+}
+
+// proxyConfigController offers a way to cache proxy configurations from
+// 3scale system.
```

Comment:
perhaps best to move this to comments at the top of the file and describe here the purpose of the fields?